File: pacientes-api/src/main/java/com/veris/pacientes_api/entity/Paciente.java
@Entity
@Table(name = "MGM_PACIENTES")
@Data
public class Paciente {

    @Id
    @Column(name = "ID_PACIENTE")
    private Long idPaciente;

    @ManyToOne
    @JoinColumn(name = "CODIGO_TIPO_IDENTIFICACION")
    private TipoIdentificacion tipoIdentificacion;

    @Column(name = "NUMERO_IDENTIFICACION")
    private String numeroIdentificacion;

    @Column(name = "PRIMER_NOMBRE")
    private String primerNombre;

    @Column(name = "SEGUNDO_NOMBRE")
    private String segundoNombre;

    @Column(name = "PRIMER_APELLIDO")
    private String primerApellido;

    @Column(name = "SEGUNDO_APELLIDO")
    private String segundoApellido;

    @Column(name = "NOMBRE_COMPLETO")
    private String nombreCompleto;

    @Column(name = "EMAIL")
    private String email;

    @Column(name = "ESTADO")
    private String estado;

    @Column(name = "FECHA_INGRESO")
    private LocalDateTime fechaIngreso;

    @Column(name = "USUARIO_INGRESO")
    private String usuarioIngreso;

    @Column(name = "FECHA_MODIFICACION")
    private LocalDateTime fechaModificacion;

    @Column(name = "USUARIO_MODIFICACION")
    private String usuarioModificacion;
}

File: backend-pacientes/src/entities/Paciente.ts
import {
  Entity,
  Column,
  PrimaryColumn,
  ManyToOne,
  JoinColumn
} from "typeorm";
import { TipoIdentificacion } from "./TipoIdentificacion";
@Entity("MGM_PACIENTES")
export class Paciente {

  @PrimaryColumn({ name: "ID_PACIENTE", type: "int" })
  idPaciente!: number;

  @Column({ name: "NUMERO_IDENTIFICACION" })
  numeroIdentificacion!: string;

  @Column({ name: "PRIMER_NOMBRE" })
  primerNombre!: string;

  @Column({ name: "SEGUNDO_NOMBRE", nullable: true })
  segundoNombre?: string;

  @Column({ name: "PRIMER_APELLIDO" })
  primerApellido!: string;

  @Column({ name: "SEGUNDO_APELLIDO", nullable: true })
  segundoApellido?: string;

  @Column({ name: "NOMBRE_COMPLETO" })
  nombreCompleto!: string;

  @Column({ name: "EMAIL" })
  email!: string;

  @Column({ name: "ESTADO" })
  estado!: string;

  @Column({ name: "FECHA_INGRESO", type: "timestamp" })
  fechaIngreso!: Date;

  @Column({ name: "USUARIO_INGRESO" })
  usuarioIngreso!: string;

  @Column({ name: "FECHA_MODIFICACION", type: "timestamp", nullable: true })
  fechaModificacion?: Date;

  @Column({ name: "USUARIO_MODIFICACION", nullable: true })
  usuarioModificacion?: string;

  @ManyToOne(() => TipoIdentificacion)
  @JoinColumn({ name: "CODIGO_TIPO_IDENTIFICACION" })
  tipoIdentificacion!: TipoIdentificacion;

}
